crossdock/server/endtoend.py

```diff
@@ -99,7 +101,7 @@ def generate_traces(self, request, response_writer):
         tracer = self.tracers[sampler_type]
         for _ in range(req.get('count', 0)):
             span = tracer.start_span(req['operation'])
-            for k, v in req.get('tags', {}).iteritems():
+            for k, v in req.get('tags', {}).items():
```

Comment:
this is not an equivalent change. `itemitems()` in 2.7 returns an iterator, while `items()` creates a full copy.